packages/reverse-exports/package.json

```diff
@@ -15,6 +15,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "mem": "^8.0.0",
```

Comment:
New versions are ESM only, which we cannot use it seems, since we are still compiling to CJS. 😟

Reply:
is switching this to memoize the only thing that "needs" to be done post rebase? 

or, does this even need to change?